src/Common/tests/System/IO/Compression/ZipTestHelper.cs

```diff
@@ -255,6 +255,8 @@ private static void ItemEqual(String[] actualList, List<FileData> expectedList,
                 String aName = Path.GetFileName(aEntry);
 
                 var bData = expectedList.Where(f => String.Equals(f.Name, aName, StringComparison.OrdinalIgnoreCase)).FirstOrDefault();
+                if (bData == null)
```

Comment:
it is causing many tests fail on OSX. we get cases there that bData will be null and then later will get null reference exception.